src/components/MovieSearch.ts

```diff
@@ -0,0 +1,59 @@
+import { BREAKPOINT_SMALL } from '../constant/movie';
+import { $ } from '../utils/Dom';
+
+export default class MovieSearch extends HTMLElement {
+  connectedCallback() {
+    this.render();
+    this.setEvent();
+  }
+
+  render() {
+    this.innerHTML = /*html*/ `
+         <div class="search-box">
+          <input class="search-input" type="text" placeholder="검색" />
+          <button class="search-button">검색</button>
+         </div>
+        `;
+  }
+
+  setEvent() {
+    const $searchButton = this.querySelector('.search-button') as HTMLElement;
+    const $searchInput = this.querySelector('input') as HTMLElement;
+
+    $searchButton.addEventListener('click', () => {
+      this.createSearchMovieEvent();
+    });
+
+    $searchInput.addEventListener('keyup', (event) => {
+      if (event.key === 'Enter') this.createSearchMovieEvent();
+    });
```

Comment:
```suggestion
    $searchButton.addEventListener('click', this.createSearchMovieEvent);
    $searchInput.addEventListener('keyup', this.createSearchMovieEvent);
```

`this.createSearchMovieEvent` 함수에 Event 인자를 주입받고, 여기에서 `if (event.key === 'Enter')`를 검사하는 것은 어떨까요?